tests/llama_stack/safety/test_trustyai_fms_provider.py

```
@@ -104,3 +105,18 @@ def test_fms_guardrails_run_shield(self, minio_pod, minio_data_connection, llama
         assert run_shields_response.violation.metadata["shield_id"] == SECURE_SHIELD_ID, (
             f"Expected shield_id to be {SECURE_SHIELD_ID}"
         )
+
+    def test_fms_moderations(self, minio_pod, minio_data_connection, llama_stack_client):
```

Ruff (0.14.4)

Unused method argument: `minio_pod`

(ARG002)